### ISSUE TITLE
error in doc: walkthrough not working

### ISSUE THREAD

**Issue by vsaase** [StanJulia/Stan.jl#83]
Hi,
just tried to walk through the walkthrough, the lines

```
data = Dict("N" => 10, "y" => [0, 1, 0, 1, 0, 0, 0, 0, 0, 1])
rc = stan_sample(sm, data)
```

raise the following error:

`ERROR: MethodError: no method matching stan_sample(::SampleModel, ::Dict{String,Any})`

**Comment by goedman** [comment]
Hi Victor,

Yes, definitely an error in the docs, should be:
```
rc = stan_sample(sm, data=data);
```

Thanks for reporting this!

**Comment by goedman** [comment]
Fixed on Stan 6.0.1
